src/main/java/org/sagebionetworks/bridge/spring/controllers/CRCController.java

```diff
@@ -697,8 +697,8 @@ private int deleteReportAndUpdateState(App app, String userId) {
             throw new EntityNotFoundException(Account.class);
         }
 
-        reportService.deleteParticipantReportRecord(app.getIdentifier(), APPOINTMENT_REPORT,
-                JAN1.toString(), account.getHealthCode());
+        reportService.deleteParticipantReportRecord(app.getIdentifier(), userId, APPOINTMENT_REPORT, JAN1.toString(),
+                account.getHealthCode());
 
         updateState(account, SELECTED);
         accountService.updateAccount(account);
```

Comment:
Josh should review this to make sure it will work.

Reply:
These calls should succeed because we do a study check and Columbia's machine account is in an organization that sponsors the study, BUT it now also needs to be a study coordinator. I made it a researcher which I'm sure will succeed.